doc/src/proxies.md

```diff
@@ -0,0 +1,23 @@
+# Proxies
+
+`rustup` provides proxy wrapper through [components](https://github.com/rust-lang/rustup/blob/master/doc/src/concepts/components.md) and you can see the list of the proxies [here](https://github.com/rust-lang/rustup/blob/22dc71adbd7e19de0090f991b3609e5f14910dcb/src/lib.rs#L17-L28)
```

Comment:
This might read slightly better as something like:

```markdown
`rustup` provides a number of wrappers for common Rust tools.
These are called _proxies_ and represent commands which are
provided by the various [components][].

The list of proxies is currently static in `rustup` and is as follows:

[components]: concepts/components.html
```

Reply:
@kinnison I appreciate your feedback. I agree that moving the doc under the `concepts/` subtree is more descriptive and clear.